bootstrap/005_table_migrations.sql

```diff
@@ -84,3 +84,34 @@ BEGIN
     call INTERNAL.UPDATE_LABEL_VIEW();
     return 'Success';
 END;
+
+
+create or replace procedure internal.generate_insert_statement_cmd(target_schema varchar, target_table varchar, source_schema varchar, source_table varchar, where_clause varchar)
+returns string
+as
+$$
+begin
+  let columns string := (
+      SELECT LISTAGG('"' || COLUMN_NAME || '"', ', ')
+      FROM (
+      SELECT COLUMN_NAME, DATA_TYPE FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_SCHEMA = :source_schema AND TABLE_NAME = :source_table
```

Comment:
Do we want to force an `ORDER BY ORDINAL_POSITION` or is that already guaranteed reading from `INFORMATION_SCHEMA.COLUMNS`?

Reply:
no longer neccessary with new algorithm for inserts